endpoints/citrus-kafka/src/test/java/org/citrusframework/kafka/endpoint/KafkaConsumerTest.java

```diff
@@ -157,6 +188,6 @@ public void testWithMessageHeaders() {
         Message receivedMessage = endpoint.createConsumer().receive(context);
         Assert.assertEquals(receivedMessage.getPayload(), controlMessage.getPayload());
         Assert.assertNotNull(receivedMessage.getHeader("Operation"));
-        Assert.assertTrue(receivedMessage.getHeader("Operation").equals("sayHello"));
+        Assert.assertEquals(receivedMessage.getHeader("Operation"), "sayHello");
```

Comment:
static imports?

Reply:
didn't want to refactor too much. but done.